src/main/resources/org/jpeek/metrics/MWE.xsl

```diff
@@ -85,7 +85,8 @@ SOFTWARE.
       </xsl:for-each>
     </xsl:variable>
     <xsl:copy>
-      <xsl:attribute name="value" select="max($OiDi/oidi/@v)"/>
+      <!--xsl:attribute name="value" select="max($OiDi/oidi/@v)"/-->
```

Comment:
@uurl what is the point of this comment? Either delete it or explain what should be done with it.